### ISSUE TITLE
[Bug] Custom Component for Node not works with markdown v5.0.1

### ISSUE THREAD

**Issue by sytolk** [Saul-Mirone/milkdown#196]
**Describe the bug**
I have use Custom Component for Node from this example: https://github.com/Saul-Mirone/milkdown/blob/main/gh-pages/pages/integrations/react/index.md#custom-component-for-node

And this works with milkdown v4.13.1 but after update to milkdown v5.0.1 not work and link Node is not replaced

```javascript
import React, { forwardRef } from 'react';
import { editorViewCtx, parserCtx } from '@milkdown/core';
import { EditorRef, ReactEditor, useEditor, useNodeCtx } from '@milkdown/react';
import { commonmark, link } from '@milkdown/preset-commonmark';

const TSLink: React.FC = ({ children }) => {
      const { node } = useNodeCtx();

      const clickLink = (evt: any) => {
        evt.preventDefault();

        let path;
        if (!hasURLProtocol(node.attrs.href)) {
          path =
            (isWeb ? '' : 'file://') +
            // @ts-ignore
            window.fileDirectory +
            '/' +
            encodeURIComponent(node.attrs.href);
        } else {
          path = node.attrs.href;
        }

        window.parent.postMessage(
          JSON.stringify({
            command: 'openLinkExternally',
            link: path
          }),
          '*'
        );
      };
      return readOnly ? (
        <a href="#" onClick={clickLink}>
          {children}
        </a>
      ) : (
        <a href="#">{children}</a>
      );
    };

const editor = useEditor(
      (root, renderReact) => {
        const nodes = commonmark
          .configure(link, { view: renderReact(TSLink) });
        return createEditor(
          root,
          md,
          readOnly,
          setEditorReady,
          nodes,
          onChange
        );
      },
      [readOnly, md, onChange]
    );
```

**Comment by sytolk** [comment]
I have try to update milkdown in example project but without success: 
https://codesandbox.io/s/milkdown-react-custom-element-forked-w14w5?file=/package.json

`core.remarkPluginFactory is not a function`

**Comment by Saul-Mirone** [comment]
Hey, there's some breaking changes in v5, I'm writing some migration guide but it take time.

I changed some code for you, wish it helps.
https://codesandbox.io/s/milkdown-react-custom-element-forked-q5v7n?file=/src/App.js

**Comment by sytolk** [comment]
Thanks for the fast fix but I have not see red color of the paragraph now. Are you sure that Node is replced with paragraph component ?

**Comment by Saul-Mirone** [comment]
> Thanks for the fast fix but I have not see red color of the paragraph now. Are you sure that Node is replaced with paragraph component ?

You are right, I think here is a bug of view options, will be fixed ASAP

**Comment by Saul-Mirone** [comment]
Fixed here: https://github.com/Saul-Mirone/milkdown/commit/5cfc30079395f0de841669c11d1a5f0aa3261605
Will be released soon.

**Comment by Saul-Mirone** [comment]
Released

**Comment by sytolk** [comment]
@Saul-Mirone thanks for the fix! I have tested with milkdown v5.1.1 and have error here:
https://codesandbox.io/s/milkdown-react-custom-element-forked-q5v7n?file=/src/App.js:763-801
```javascript
import { useNodeCtx } from '@milkdown/react';

const { node } = useNodeCtx();
node.attrs.src // -> node.attrs is undefned here TypeError: Cannot read properties of undefined (reading 'src')
```

**Comment by Saul-Mirone** [comment]
> @Saul-Mirone thanks for the fix! I have tested with milkdown v5.1.1 and have new error here: https://codesandbox.io/s/milkdown-react-custom-element-forked-q5v7n?file=/src/App.js:763-801
> 
> ```js
> import { useNodeCtx } from '@milkdown/react';
> 
> const { node } = useNodeCtx();
> node.attrs.src // -> node.attrs is undefned here TypeError: Cannot read properties of undefined (reading 'src')
> ```
> 
> Debugging show that node is empty object for me `node = {}`

Havn't seen any errors in the codeSandBox, are you sure you update the configure from `view: renderReact(CustomParagraph)` to `view: () => renderReact(CustomParagraph)` ?
